licenses/library.go

```diff
@@ -80,9 +80,13 @@ func Libraries(ctx context.Context, classifier Classifier, importPaths ...string
 	pkgsByLicense := make(map[string][]*packages.Package)
 	errorOccurred := false
 	packages.Visit(rootPkgs, func(p *packages.Package) bool {
-		if len(p.Errors) > 0 {
-			errorOccurred = true
-			return false
+		for _, err := range p.Errors {
+			// Ignore errors complaining that the package is a binary. We still want
+			// to traverse related packages.
+			if !strings.Contains(err.Msg, "is a program, not an importable package") {
+				errorOccurred = true
+				return false
+			}
```

Comment:
This seems like it should be a separate PR (all of the other changes are related to build tags, but this seems unrelated).